### PR TITLE
added guide on setting up a local domain for use with ingress

### PR DESCRIPTION
**Pull request by mjpitz**
This walks users through setting up a local domain on their machine to make testing ingress easier.

### CODE REVIEW

File: site/content/docs/user/resources.md [hunk 1]
```diff
@@ -71,6 +72,9 @@ https://mauilion.dev/posts/kind-k8s-testing/
 
 https://projectcontour.io/kindly-running-contour/
 
+### Setting up a local domain for use with ingress
```
Comment (aojea) [inline site/content/docs/user/resources.md:75]: the blog title sounds a better fit
```
Local Ingress Domains for your Kind Cluster
```

File: site/content/docs/user/resources.md [hunk 2]
```diff
@@ -71,6 +72,9 @@ https://mauilion.dev/posts/kind-k8s-testing/
 
 https://projectcontour.io/kindly-running-contour/
 
+### Local Ingress Domains for your Kind Cluster
+
+- OSX: https://mjpitz.com/blog/2020/10/21/local-ingress-domains-kind/
```
Comment (aojea) [inline site/content/docs/user/resources.md:77]: this does not render ok, you can see it here

https://deploy-preview-1908--k8s-kind.netlify.app/docs/user/resources/

maybe just remove the `- OSX:` part and add only the url, so it is consistent with the previous sections ... or add the OSX part to the section title if you want to highlight the OS

**Comment by aojea** [comment]
/lgtm
/assign @BenTheElder 
Thanks

**Comment by BenTheElder** [comment]
/lgtm
/approve
thanks!

**Pull request merged**